TemplateProject/fastlane/Fastfile

```diff
@@ -0,0 +1,95 @@
+fastlane_version "2.44.1"
```

Comment:
I think it's fine for individual projects to lock this down to a version, but my concern is that by locking the _template_ down, it means we will need to keep this up to date or else we will be creating templates with out of date versions of Fastlane by default. That seems to me like the wrong behavior. I'd also probably advocate for ignoring the template Gemfile.lock for the same reason.